modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/OrganizationResourceTest.java

```diff
@@ -762,6 +792,20 @@ protected Organization testPutOrganization_addOrganization()
 			randomOrganization());
 	}
 
+	private AccountEntry _addAccountEntry() throws Exception {
+		AccountEntry accountEntry = _accountEntryLocalService.addAccountEntry(
+			TestPropsValues.getUserId(),
+			AccountConstants.ACCOUNT_ENTRY_ID_DEFAULT,
+			RandomTestUtil.randomString(), RandomTestUtil.randomString(), null,
+			null, null, null, AccountConstants.ACCOUNT_ENTRY_TYPE_BUSINESS,
+			WorkflowConstants.STATUS_APPROVED,
+			ServiceContextTestUtil.getServiceContext());
+
+		accountEntry.setExternalReferenceCode(RandomTestUtil.randomString());
```

Comment:
it should already put a random string the the erc. Why do we need this?

Reply:
You are right also in the classes
```
AccountResourceTest
UserAccountResourceTest
```
the same thing is done, I modify

Reply:
I looked at the code and the `addAccountEntry` method does not populate the `externalReferenceCode`, do I replace with the `addOrUpdateAccountEntry` method?